caas/kubernetes/provider/k8s_test.go

```diff
@@ -6752,11 +6752,12 @@ func (s *K8sBrokerSuite) TestEnsureServiceWithConstraints(c *gc.C) {
 	}
 	for i := range podSpec.Containers {
 		podSpec.Containers[i].Resources = core.ResourceRequirements{
-			Limits: core.ResourceList{
+			Requests: core.ResourceList{
```

Comment:
So we change operator caas app to use `Requests` as well

Reply:
yeah, Requests is the better approach